packages/patternfly-4/src/pages/index.js

```diff
@@ -1,20 +1,215 @@
+                <NavList className="pf-c-nav__list-footer">
+                  <NavItem to="#" aria-label="Get started with PatternFly 4">
+                    Get started
+                  </NavItem>
+                  <NavItem to="#" aria-label="PatternFly 4 components">
+                    Components
+                  </NavItem>
+                  <NavItem to="#" aria-label="PatternFly 4 layouts">
+                    Layouts
+                  </NavItem>
+                  <NavItem to="#" aria-label="PatternFly 4 styles">
+                    Styles
+                  </NavItem>
+                  <NavItem to="#" aria-label="View the PatternFly 3 website">
+                    PatternFly 3
+                  </NavItem>
+                </NavList>
+              </Nav>
+            </GridItem>
+            <GridItem span={6} sm={4}>
+              <Title size="md" className="pf-m-white">CONTRIBUTE</Title>
+              <Nav aria-label="Quick Links">
+                <NavList className="pf-c-nav__list-footer">
+                  <NavItem to="#" aria-label="How to contribute as a Designer">
+                    Designers
+                  </NavItem>
+                  <NavItem to="#" aria-label="How to contribute as a Developer">
+                    Developers
+                  </NavItem>
+                  <NavItem to="#" aria-label="PatternFly 4 Code of Conduct">
+                    Code of Conduct
+                  </NavItem>
+                </NavList>
+              </Nav>
+            </GridItem>
+          </Grid>
+        </GridItem>
+        <GridItem md={5}>
+          <Grid className="pf-u-py-xl-on-sm pf-u-py-8-on-md">
+            <GridItem md={7} mdOffset={5} className="pf4-l-footer-column pf-u-p-lg">
+              <img src={patternflyLogo} alt="PatternFly logo" className="pf-u-pb-lg" />
+              <p className="pf-m-white pf-u-pb-lg">
+                PatternFly is an open source design system built to drive consistency and unify teams. We provide tools like design documentation, components, and code examples to make it possible for anyone to design and build responsive, accessible web applications.
+              </p>
+              <Link to="https://www.facebook.com/groups/PatternFlyCommunity/" target="top" aria-label="Link to PatternFly Facebook page">
+                <i class="fab fa-facebook"></i>
+              </Link>
+              <Link to="https://github.com/patternfly/patternfly" target="top" aria-label="Link to PatternFly GitHub page">
+                <i class="fab fa-github"></i>
+              </Link>
+              <Link to="https://twitter.com/patternfly_des" target="top" aria-label="Link to PatternFly Twitter page">
+                <i class="fab fa-twitter"></i>
+              </Link>
+            </GridItem>
+          </Grid>
+        </GridItem>
+      </Grid>
+    </PageSection>
+    <PageSection className="pf4-l-footer-dark">
```

Comment:
Can you pull the footer and related imports out into a separate component so we can reuse it

Reply:
I can give it a shot - not sure that the other pages will have the footer though - I haven't seen it in any of the other mocks.

Reply:
I think most pages have the footer on every page @rachael-phillips

Reply:
@jschuler I've taken the footer out of the `index.js` page and created a separate component. The site renders without errors, so I think it is all set?